src/plugins/mf_priority.cpp

```diff
@@ -184,6 +204,71 @@ static int priority_cb (flux_plugin_t *p,
         return -1;
     }
 
+    b = static_cast<bank_info *> (flux_jobtap_job_aux_get (
+                                                    p,
+                                                    FLUX_JOBTAP_CURRENT_JOB,
+                                                    "mf_priority:bank_info"));
+
+    if (b == NULL) {
+        flux_jobtap_raise_exception (p, FLUX_JOBTAP_CURRENT_JOB, "mf_priority",
+                                     0, "job.state.priority: bank info is " \
+                                     "missing");
+
+        return -1;
+    }
+
+    std::map<int, std::map<std::string, struct bank_info>>::iterator it;
+    std::map<std::string, struct bank_info>::iterator bank_it;
+
+    if (b->max_run_jobs == BANK_INFO_MISSING) {
+        // try to look up user again
+        it = users.find (userid);
+        if (it == users.end ()) {
+            return flux_jobtap_priority_unavail (p, args);
+        } else {
+            // make sure user belongs to bank they specified; if no bank was
+            // passed in, look up their default bank
+            if (bank != NULL) {
+                bank_it = it->second.find (std::string (bank));
+                if (bank_it == it->second.end ())
+                    flux_jobtap_raise_exception (p, FLUX_JOBTAP_CURRENT_JOB,
+                                                 "mf_priority",
+                                                 0,
+                                                 "job.state.priority: user " \
+                                                 "does not belong to " \
+                                                 "specified bank");
+            } else {
+                bank = const_cast<char*> (users_def_bank[userid].c_str ());
+                bank_it = it->second.find (std::string (bank));
+                if (bank_it == it->second.end ())
+                    flux_jobtap_raise_exception (p, FLUX_JOBTAP_CURRENT_JOB,
+                                                 "mf_priority",
+                                                 0,
+                                                 "job.state.priority: " \
+                                                 "user/default bank entry " \
+                                                 "does not exist");
```

Comment:
similar comments to above: no need for line continuation and generalize error message.